client/public/index.html

```diff
@@ -21,6 +21,11 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
+     <!-- Google Tag Manager -->
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+      new Date().getTime(),event:'gtm.js'});})(window,document,'script','dataLayer','GTM-NMPT5JP');</script>
+    <!-- End Google Tag Manager -->
```

Comment:
This looks good but I wonder if we should just simplify it so it's more readable:

```js
window.dataLayer = window.dataLayer || [];
window.dataLayer.push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
```

It might also be worth adding a comment mentioning that loading of the actual GTM script is done in React so that `<title>` is properly set by the time GTM loads, just so future readers won't get too confused?  I dunno.

Reply:
For sure